model-optimizer/extensions/middle/SplitConcatPairToInterpolate.py

```diff
@@ -53,6 +53,9 @@ def get_concat_after_split(split: Node) -> Optional[Node]:
 
 
 def get_interpolate_pattern(split: Node) -> dict:
+    split_shape = split.in_port(0).data.get_shape()
+    if split_shape is None or len(split_shape) < 4 or len(split_shape) > 5:
```

Comment:
Why do you check that the `split_shape is None`? Did you face that case? If so that means that some other transformation did not work correctly and it is necessary to investigate the reason for that.

Reply:
Fixed.